### ISSUE TITLE
Worldwide location or multiple locations for InterestOverTime

### ISSUE THREAD

**Issue by FatherFil** [gabrielfs7/google-trends#13]
Is it possible to add the ability to perform a "Worldwide" location search, or even a multiple location search?

At the moment ```->withLocation("")``` defaults to US, or you have to supply a single country. It would be great to be able to specify the Worldwide view, or multiple locations if not.

**Comment by gabrielfs7** [comment]
Yes, could be possible, but it would require multiple queries on Google api, cause even on Google Trends site it is not possible to select multiple countries. I accept help to develop this :) Good idea!

**Comment by RobinE89** [comment]
"cause even on Google Trends site it is not possible to select multiple countries" not multiple countries -> BUT Worldwide is possible on GoogleTrends itself. I think the global overview is actually the most important setting that is apparently missing here? that way I can't use the api....
